packages/select/src/vaadin-lit-select.js

```diff
@@ -41,12 +41,16 @@ class Select extends SelectBaseMixin(ElementMixin(ThemableMixin(PolylitMixin(Lit
       inputFieldContainer,
       screenReaderOnly,
       css`
-        :host {
-          position: relative;
+        ::slotted([slot='value']) {
+          flex: 1;
```

Comment:
IIRC setting `flex-grow` should be enough, no need to change this.

Reply:
Shouldn't it shrink as well if needed?

Reply:
Yes, makes sense.

Reply:
At least it doesn't work as expected if I change this back to `flex-grow: 1`.